regulations/build/regulations/history/official/2019-05-01/guidelines.html.erb

```diff
@@ -0,0 +1,240 @@
+<% provide(:title, 'WCA Guidelines') %>
+<div class="container"><h1>WCA Guidelines</h1>
+<div class="version">Version: May 1, 2019<br/>[<code><a href="https://github.com/thewca/wca-regulations/tree/official/">official</a>:<a href="https://github.com/thewca/wca-regulations/commits/unknown">unknown</a></code>]</div>
```

Comment:
It's missing the commit hash and the usual disclaimer that we manually add that these are not the official regulations, see [here](https://github.com/thewca/worldcubeassociation.org/blob/regulations-data/regulations/build/regulations/history/official/2018-01-01/index.html.erb#L3) for instance.
For the commit hash you can specify it with `-g 72bccb9` (which should be the correct commit if I'm not mistaken), and then you will have to add the comment by editing the file manually, we don't have a tool for that yet :s

Reply:
Yes, is `-g 72ccb9`. I'll add the disclaimer right now :)